### PR TITLE
Pints

### LINKED ISSUE #39: Support both NT and TSS assays
## NT
- homer
- grohmm

## TSS
- PINTS

### PR DESCRIPTION
**Pull request by Emiller88**
Closes #39

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -126,26 +127,47 @@ process {
...
     withName: GROHMM_TRANSCRIPTCALLING {
-        ext.when = { params.transcript_identification == "grohmm" }
+        ext.when = { !params.skip_grohmm && params.assay_type == "GROseq" }
     }
 
     withName: HOMER_FINDPEAKS {
-        ext.when = { params.transcript_identification == "homer" }
-        ext.args = "-style factor"
+        ext.when = { params.assay_type == "GROseq" }
+        // FIXME Handle other assays
```
Comment (jfy133) [inline conf/modules.config:139]: TODO is more typical, or do you want to distinguish with nf-core TODO?

File: conf/test.config [hunk 2]
```diff
@@ -20,10 +20,21 @@ params {
...
+    input = "${projectDir}/assets/samplesheet.csv"
 
     // Genome references
     fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/nascent/reference/GRCh38_chr21.fa'
     gtf = 's3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.gtf'
+
+    assay_type = "GROseq"
+    skip_grohmm = true
     tuning_file = "${projectDir}/tests/subworkflows/local/grohmm/tuningparams_small.csv"
+    filter_bed = "${projectDir}/tests/samplesheets/region.bed"
```
Comment (jfy133) [inline conf/test.config:32]: Is this a file that would be used for all users, or could it be moved to test-datasets given this is in the `test.config`?

File: nextflow_schema.json [hunk 3]
```diff
@@ -66,22 +66,40 @@
...
+                        "STRIPEseq",
+                        "PROseq",
+                        "GROseq",
+                        "R_5",
+                        "R_3",
+                        "R1_5",
+                        "R1_3",
+                        "R2_5",
+                        "R2_3"
+                    ]
```
Comment (jfy133) [inline nextflow_schema.json:91]: Do these relate to the FIXME comments above? Does it make sense to have the full list already if not all are supported (or is this for something else?)

File: subworkflows/local/transcript_identification.nf [hunk 4]
```diff
@@ -0,0 +1,73 @@
+include { GROHMM                } from './grohmm'
```
Comment (jfy133) [inline subworkflows/local/transcript_identification.nf:1]:
```suggestion
include { GROHMM } from './grohmm'
```

File: subworkflows/local/transcript_identification.nf [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    ch_versions = ch_versions.mix(BEDTOOLS_INTERSECT_FILTER.out.versions.first())
+
+    emit:
+    grohmm_td_plot = GROHMM.out.td_plot
+    homer_peaks = HOMER_GROSEQ.out.peaks
+    homer_tagdir = HOMER_GROSEQ.out.tagdir
+
+    transcript_beds = ch_identification_bed_clean
+
+    versions = ch_versions
```
Comment (jfy133) [inline subworkflows/local/transcript_identification.nf:72]: Missing multiqc emission

**Review by jfy133 — APPROVED** [review]

File: conf/modules.config (hunk 1 above)
Comment (Emiller88) [reply conf/modules.config:139]: Ah, it's the next line, I have the `-style groseq` hardcoded. Outside the scope, but I agree, it's a todo!

File: conf/test.config (hunk 2 above)
Comment (Emiller88) [reply conf/test.config:32]: Yes, but it'll also work from here, and that way it's quick to update.

File: nextflow_schema.json (hunk 3 above)
Comment (Emiller88) [reply nextflow_schema.json:91]: They're supported in PINTS, but I don't have homer support for example.

**Pull request merged**